### PR TITLE
README, polishing and updating all missing contents

### PR DESCRIPTION
**Pull request by guzzilar**
### 1. Objective

There are bunch of things happened, added, and removed.
This pull request is to document all the missing contents to README.md file

### 2. Description of change

- Adding a new content under the Installation Instruction section, **"WordPress Plugin Manager (recommended)"**.
Screen Shot 2562-11-06 at 05 37 34 copy

- Updating the Getting Started's index accordingly (also shorten those url).
Screen Shot 2562-11-06 at 05 40 00 copy

- Replacing 3 screenshots under the Manual Installation section.
Screen Shot 2562-11-06 at 05 42 14

- Replacing a screenshot of Omise Setting page (under **"First Time Setup"** section).
Screen Shot 2562-11-06 at 05 43 44

- Under **"First Time Setup"** section, step [3]. Adding a link to Omise Dashboard, **"Keys"** page.
Please note that this link will open a test key page (https://dashboard.omise.co/test/keys)
Screen Shot 2562-11-06 at 05 45 01 copy

- Correcting the available payment method list (adding TrueMoney Wallet).
Screen Shot 2562-11-06 at 05 47 03 copy

- Removing **"What's next"** section.
As I hate to do it, but we don't have a better place to document all the useful features of this plugin. So I'm removing it out for now.
Screen Shot 2562-11-06 at 05 49 30 copy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,27 +60,27 @@ Next, check **[First Time Setup](#first-time-setup)** to continue setting up you
...
+3. At the Omise Settings page, you are going to set your `Public key` and `Secret key` (these keys can be found at [Omise Dashboard](https://dashboard.omise.co/test/keys)).
 
 4. Make sure that the option `Test mode` is checked and set your Omise keys at `Public key for test` and `Secret key for test` fields if you would like to test Omise service integration.
 
 5. Click **'Save Settings'**.
 
 #### Enable payment methods
 
-> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants with a Thai-registered Omise account.
```
Comment (danfowler) [inline README.md]: I'd like to move away from "Thai-registered" to "registered in Thailand".  But this is personal preference.

File: README.md [hunk 2]
```diff
@@ -44,27 +60,27 @@ Next, check **[First Time Setup](#first-time-setup)** to continue setting up you
...
 
-There are 5 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, and **Internet Banking**.
+There are 6 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, **Internet Banking**, and **TrueMoney Wallet**.
```
Comment (danfowler) [inline README.md]: Just for the sake of laziness, I'd avoid writing the number "6" explicitly here as it's inevitable that (1) it's going to change (2) we're going to forget to update it.

File: README.md [hunk 3]
```diff
@@ -12,30 +12,46 @@ If the version you are currently using has not been listed, you can try installi
...
+You can simply install Omise WooCommerce plugin via WordPress Plugin Manager by the following 4 steps
+1. From the left sidebar at your WordPress Admin page, under **"Plugin"** section. Click **"Add New"**.
+
+2. At the **"Add Plugins"** page, search for Omise WooCommerce plugin using keyword: `Omise`.
+
+3. Click **"Install Now"** to download and install Omise WooCommerce plugin into your WordPress website.
+
+![Screen_Shot_2562-10-01_at_08 28 27_copy](https://user-images.githubusercontent.com/2154669/68250269-274f1080-0053-11ea-8db1-bab9cc32ea46.png)
+
+4. After the plugin has been downloaded and installed. The **"Install Now"** button will now be changed to `Activate`. Make sure to click **"Activate"**
```
Comment (danfowler) [inline README.md]: This one button is expressed in this document three ways: 

```
`Activate`
**"Activate"**
`activate`
```

Of the three, I prefer the middle one (I'm agnostic on the bold).

File: README.md (hunk 1 above)
Comment (guzzilar) [reply README.md]: >  Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants  that is registered in Thailand.

@danfowler  Sounds good?

File: README.md (hunk 2 above)
Comment (guzzilar) [reply README.md]: Removing the number sounds good. But cannot think of any better sentence here at the moment... 
@danfowler any suggestion? 🆘

File: README.md [hunk 4]
```diff
@@ -12,30 +12,46 @@ If the version you are currently using has not been listed, you can try installi
 
 ## Getting Started
 
-- [Installation Instructions](https://github.com/omise/omise-woocommerce/tree/improve-readme#installation-instructions)
-  - [Manually](https://github.com/omise/omise-woocommerce/tree/improve-readme#manually)
-- [First Time Setup](https://github.com/omise/omise-woocommerce/tree/improve-readme#first-time-setup)
-  - [Connect your store with your Omise account](https://github.com/omise/omise-woocommerce/tree/improve-readme#connect-your-store-with-your-omise-account)
-  - [Enable payment methods](https://github.com/omise/omise-woocommerce/tree/improve-readme#enable-payment-methods)
+- [Installation Instructions](#installation-instructions)
+  - [WordPress Plugin Manager (recommended)](#wordpress-plugin-manager-recommended)
```
[the remark this answers is not in the document]
Comment (guzzilar) [reply README.md]: Good point! 💯

File: README.md (hunk 1 above)
Comment (jonrandy) [reply README.md]: '...merchants that are registered in Thailand'

File: README.md (hunk 2 above)
Comment (jonrandy) [reply README.md]: The payment methods available for you to enable on your store are: blah, blah, blah, and blah.

File: README.md [hunk 5]
```diff
@@ -44,27 +60,27 @@ Next, check **[First Time Setup](#first-time-setup)** to continue setting up you
...
 #### Enable payment methods
 
-> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants with a Thai-registered Omise account.
 
-There are 5 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, and **Internet Banking**.
+There are 6 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, **Internet Banking**, and **TrueMoney Wallet**.
 
-1. At Omise Settings page, Payment Methods section, there will be 5 available payment methods show on the list.
+1. At Omise Settings page, Payment Methods section, there will be 6 available payment methods show on the list.
```
Comment (jonrandy) [inline README.md]: Replace 6 with 'a number of'
'show on the list' should be 'shown on the list'
'At Omise Settings page' should probably be  'At the Omise Settings page'

File: README.md [hunk 6]
```diff
@@ -44,27 +60,27 @@ Next, check **[First Time Setup](#first-time-setup)** to continue setting up you
...
 
 2. Click `config` at the table on a payment method that you would like to accept payment with.
```
Comment (jonrandy) [inline README.md]: Click `config` in the table for the payment method that you would like to accept payment with.

**Pull request merged**